Fix:

error: implicit declaration of function 'atoi' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'nefu_read_config' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- libtest/http.c.orig	2006-09-08 10:00:31.000000000 -0500
+++ libtest/http.c	2024-07-17 21:08:20.000000000 -0500
@@ -6,6 +6,7 @@
 #include <errno.h>
 #include <ctype.h>
 #include <unistd.h>
+#include <stdlib.h>
 #include <strings.h>
 #include <string.h>
 #include <stdio.h>
--- nefu.h.orig	2006-11-10 13:32:12.000000000 -0600
+++ nefu.h	2024-07-17 21:11:08.000000000 -0500
@@ -226,3 +226,7 @@
 int		machine_errors( void );
 void		machine_rcode( struct machine *, struct rcode * );
 void		machine_rcode_assign( struct machine *, struct rcode * );
+
+
+/* parse.c */
+int		nefu_read_config( char * );
